wallet/builder.go

```diff
@@ -0,0 +1,63 @@
+package wallet
+
+import (
+	"math/big"
+	"sort"
+
+	"go.sia.tech/siad/types"
+)
+
+// BytesPerInput is the encoded size of a SiacoinInput and corresponding
+// TransactionSignature, assuming standard UnlockConditions.
+const BytesPerInput = 241
+
+// DistributeFunds is a helper function for distributing the value in a set of
+// utxos among the given number of outputs, each containing the requested amount
+// of siacoins. It returns the minimal set of utxos that will fund such a
+// transaction, along with the resulting fee and change. Inputs with value equal
+// to per are ignored. If the inputs are not sufficient to fund all outputs,
+// DistributeFunds returns nil.
+func DistributeFunds(utxos []SiacoinElement, outputs int, amount, feePerByte types.Currency) ([]SiacoinElement, types.Currency, types.Currency) {
+	// desc sort
+	sort.Slice(utxos, func(i, j int) bool {
+		return utxos[i].Value.Cmp(utxos[j].Value) > 0
+	})
+
+	// estimate the fees
+	const bytesPerOutput = 64 // approximate; depends on currency size
```

Comment:
Should probably also be a constant we can reuse. 

(nit) maybe 58 is good enough? That's the size of an output that spends math.MaxUint64 Siacoins which is around 20 trillion USD right now if I'm not mistaken.

Reply:
post-utreexo this will be exactly 48 every time. :) For now, 64 is fine; it's a bit higher than strictly necessary, but fees are tiny anyway.